input/drivers_hid/wiiusb_hid.c

```diff
@@ -270,6 +270,44 @@ static int wiiusb_hid_removal_cb(int result, void *usrdata)
    return 0;
 }
 
+static bool isRetrodeGamepad(usb_devdesc devdesc)
+{
+    if (devdesc.idVendor != VID_RETRODE || devdesc.idProduct != PID_RETRODE)
+    {
+        return false;
+    }
+    if (devdesc.configurations != NULL)
+    {
+        if (devdesc.configurations->interfaces != NULL)
+        {
+            if (devdesc.configurations->interfaces->endpoints != NULL)
+            {
+                return devdesc.configurations->interfaces->bInterfaceSubClass == 0;
+            }
+        }
+    }
+    return false;
+}
+
+static bool isRetrodeMouse(usb_devdesc devdesc)
+{
+    if (devdesc.idVendor != VID_RETRODE || devdesc.idProduct != PID_RETRODE)
+    {
```

Comment:
When a block consists of just one line, get rid of the brackets.

Reply:
No problem. But now it's obvious that we can write this in one expression... (didn't do it, can't test again today)